ISLE/isleapp.cpp

```diff
@@ -379,6 +385,27 @@ SDL_AppResult SDL_AppIterate(void* appstate)
 		if (g_mousemoved) {
 			g_mousemoved = FALSE;
 		}
+
+		if (g_lastJoystickMouseX != 0 || g_lastJoystickMouseY != 0) {
```

Comment:
Is there a reason not to put this code in the event handler directly?

Reply:
SDL_AppEvent gives event only when the joystick value is changed and not when it's reporting the same value
So for example if you hold your joystick to the most top and stay like that it will not fire the event

Reply:
I see, so this is only for simulating mouse motion events based on the state of the joystick?

Let's please take this piece of code and put it into its own function at the bottom of `isleapp.cpp`, and add a comment that it is doing that. This is the only instance where we generate an event for isle that doesn't directly correctly to an event we received from SDL so it is a bit special.

Reply:
Got it, what should be the function name? `MoveVirtualCursorViaJoystick`?